Northstar.CustomServers/mod/scripts/vscripts/gamemodes/_gamemode_fd.nut

```diff
@@ -311,211 +314,212 @@ void function mainGameLoop()
-	npcs[eFD_AITypeIDs.TITAN_ARC]<-0
-	npcs[eFD_AITypeIDs.TITAN_MORTAR]<-0
-	npcs[eFD_AITypeIDs.GRUNT]<-0
-	npcs[eFD_AITypeIDs.SPECTRE]<-0
-	npcs[eFD_AITypeIDs.SPECTRE_MORTAR]<-0
-	npcs[eFD_AITypeIDs.STALKER]<-0
-	npcs[eFD_AITypeIDs.REAPER]<-0
-	npcs[eFD_AITypeIDs.TICK]<-0
-	npcs[eFD_AITypeIDs.DRONE]<-0
-	npcs[eFD_AITypeIDs.DRONE_CLOAK]<-0
-	// npcs[eFD_AITypeIDs.RONIN]<-0
-	// npcs[eFD_AITypeIDs.NORTHSTAR]<-0
-	// npcs[eFD_AITypeIDs.SCORCH]<-0
-	// npcs[eFD_AITypeIDs.LEGION]<-0
-	// npcs[eFD_AITypeIDs.TONE]<-0
-	// npcs[eFD_AITypeIDs.ION]<-0
-	// npcs[eFD_AITypeIDs.MONARCH]<-0
-	// npcs[eFD_AITypeIDs.TITAN_SNIPER]<-0
-
-
-	foreach(WaveEvent e in waveEvents[waveIndex])
+	npcs[eFD_AITypeIDs.TITAN] <- 0
+	npcs[eFD_AITypeIDs.TITAN_NUKE] <- 0
+	npcs[eFD_AITypeIDs.TITAN_ARC] <- 0
+	npcs[eFD_AITypeIDs.TITAN_MORTAR] <- 0
+	npcs[eFD_AITypeIDs.GRUNT] <- 0
+	npcs[eFD_AITypeIDs.SPECTRE] <- 0
+	npcs[eFD_AITypeIDs.SPECTRE_MORTAR] <- 0
+	npcs[eFD_AITypeIDs.STALKER] <- 0
+	npcs[eFD_AITypeIDs.REAPER] <- 0
+	npcs[eFD_AITypeIDs.TICK] <- 0
+	npcs[eFD_AITypeIDs.DRONE] <- 0
+	npcs[eFD_AITypeIDs.DRONE_CLOAK] <- 0
+	// npcs[eFD_AITypeIDs.RONIN] <- 0
+	// npcs[eFD_AITypeIDs.NORTHSTAR] <- 0
+	// npcs[eFD_AITypeIDs.SCORCH] <- 0
+	// npcs[eFD_AITypeIDs.LEGION] <- 0
+	// npcs[eFD_AITypeIDs.TONE] <- 0
+	// npcs[eFD_AITypeIDs.ION] <- 0
+	// npcs[eFD_AITypeIDs.MONARCH] <- 0
+	// npcs[eFD_AITypeIDs.TITAN_SNIPER] <- 0
+
+
+	foreach( WaveEvent e in waveEvents[waveIndex] )
 	{
-		if(e.spawnEvent.spawnAmount==0)
+		if( e.spawnEvent.spawnAmount == 0 )
 			continue
-		switch(e.spawnEvent.spawnType)
+		switch( e.spawnEvent.spawnType )
 		{
-			case(eFD_AITypeIDs.TITAN):
-			case(eFD_AITypeIDs.RONIN):
-			case(eFD_AITypeIDs.NORTHSTAR):
-			case(eFD_AITypeIDs.SCORCH):
-			case(eFD_AITypeIDs.TONE):
-			case(eFD_AITypeIDs.ION):
-			case(eFD_AITypeIDs.MONARCH):
-			case(eFD_AITypeIDs.LEGION):
-			case(eFD_AITypeIDs.TITAN_SNIPER):
+			case( eFD_AITypeIDs.TITAN ):
+			case( eFD_AITypeIDs.RONIN ):
+			case( eFD_AITypeIDs.NORTHSTAR ):
+			case( eFD_AITypeIDs.SCORCH ):
+			case( eFD_AITypeIDs.TONE ):
+			case( eFD_AITypeIDs.ION ):
+			case( eFD_AITypeIDs.MONARCH ):
+			case( eFD_AITypeIDs.LEGION ):
+			case( eFD_AITypeIDs.TITAN_SNIPER ):
 				npcs[eFD_AITypeIDs.TITAN]+=e.spawnEvent.spawnAmount
 				break
 			default:
 				npcs[e.spawnEvent.spawnType]+=e.spawnEvent.spawnAmount
 		}
 	}
-	array<int> ret = [-1,-1,-1,-1,-1,-1,-1,-1,-1]
-	foreach(int key,int value in npcs)
+	array<int> ret = [ -1, -1, -1, -1, -1, -1, -1, -1, -1 ]
+	foreach( int key, int value in npcs )
 	{
-		if(value==0)
+		if( value == 0 )
 			continue
 		int lowestArrayIndex = 0
 		bool keyIsSet = false
-		foreach(index,int arrayValue in ret)
+		foreach( index, int arrayValue in ret )
 		{
-			if(arrayValue==-1)
+			if( arrayValue == -1 )
 			{
 				ret[index] = key
 				keyIsSet = true
 				break
 			}
-			if(npcs[ret[lowestArrayIndex]]>npcs[ret[index]])
+			if( npcs[ret[lowestArrayIndex]] > npcs[ret[index]] )
```

Comment:
```suggestion
			if( npcs[ ret[ lowestArrayIndex ] ] > npcs[ ret[ index ] ] )
```
I'd say for nested [] we should use spaces for better readability

Reply:
i think the inner ones should stay without

Reply:
yeah maybe
```suggestion
			if( npcs[ ret[lowestArrayIndex] ] > npcs[ ret[index] ] )
```